.mailmap

```diff
@@ -11,3 +11,7 @@ Tamio-Vesa Nakajima <tamiove@amazon.com> <tnk@amazon.com>
 Iulian Barbu <iul@amazon.com>
 Petre Eftime <epetre@amazon.com> <petre.eftime@gmail.com>
 karthik nedunchezhiyan <karthik.n@zohocorp.com> <karthik1705.n@gmail.com>
+Alin Dima <alindima@amazon.com>
+Andrei Sandu <sandreim@amazon.com> <54316454+sandreim@users.noreply.github.com>
```

Comment:
I think we should remove the bogus <54316454+sandreim@users.noreply.github.com> email address.

Reply:
if we do remove it, you will appear twice in the credits.md file, once for your real email address and once for this new one